src/parser/shaman/restoration/modules/features/MasteryEffectiveness.js

```diff
@@ -28,7 +28,7 @@ class MasteryEffectiveness extends Analyzer {
   masteryHealEvents = [];
 
   on_byPlayer_heal(event) {
```

Comment:
Convert to action listener... we hate magic functions

Reply:
no

Reply:
Okay